src/plugins/autobank/MQ2AutoBank.cpp

```diff
@@ -0,0 +1,431 @@
+/*
+ * MacroQuest: The extension platform for EverQuest
+ * Copyright (C) 2002-2022 MacroQuest Authors
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License, version 2, as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+
+#include <mq/Plugin.h>
+
+PreSetup("MQ2AutoBank");
+PLUGIN_VERSION(0.1);
+
+namespace mq {
+
+	static std::list<ItemGlobalIndex> gAutoBankList;
+	static std::list<ItemGlobalIndex> gAutoInventoryList;
+	static bool gbStartAutoBanking = false;
+	static bool gbAutoBankInProgress = false;
+	static bool gbAutoInventoryInProgress = false;
+	static bool gbAutoBankTradeSkillItems = false;
+	static bool gbAutoBankTrophiesWithTradeskill = true;
+	static bool gbAutoBankCollectibleItems = false;
+	static bool gbAutoBankQuestItems = false;
+	static bool gbAutoInventoryItems = false;
+
+	static CContextMenu* AutoBankMenu = nullptr;
+	static CButtonWnd* gAutoBankButton = nullptr;
+	static int s_bankCustomMenu = 0;
+
+	// BankWnd context menu items
+	constexpr int ContextMenu_TradeskillItemsId = 50;
+	constexpr int ContextMenu_CollectibleItemsId = 51;
+	constexpr int ContextMenu_QuestItemsId = 52;
+	constexpr int ContextMenu_CheckedItemsId = 53;
+	constexpr int ContextMenu_TrophysId = 54;
+}
+
+namespace AutoBank {
+	class BankWnd_Hook;
+}
+
+// CBankWnd hooks
+class AutoBank::BankWnd_Hook {
```

Comment:
style: Braces for class and function should be on next line

Reply:
Are you expecting this for classes/functions only or is this meant to be for the entirety of the plugin?

Reply:
entirety. check the original source if you're not sure.

Reply:
brackets were moved.